recipe-server/client/control/components/recipes/ImportExperimentsPage.js

```diff
@@ -0,0 +1,257 @@
+import React, { PropTypes as pt } from 'react';
+import autobind from 'autobind-decorator';
+import { Row, Col, Button, Card, Modal } from 'antd';
+import { connect } from 'react-redux';
+
+
+import LoadingOverlay from 'control/components/common/LoadingOverlay';
+import QueryExperiments from 'control/components/data/QueryExperiments';
+import { importExperiment, selectRejectedExperiment, rejectExperiment } from 'control/state/app/experimenter/actions';
+import { getExperiments, getRejectedExperiment } from 'control/state/app/experimenter/selectors';
+
+
+@connect(null, { importExperiment })
+@autobind
+export class ExperimentAcceptButton extends React.PureComponent {
+  static propTypes = {
+    importExperiment: pt.func.isRequired,
+    experiment: pt.object.isRequired,
+  };
+
+  handleClick() {
+    this.props.importExperiment(this.props.experiment);
+  }
+
+  render() {
+    return (
+      <Button type="primary" onClick={this.handleClick}>Accept</Button>
+    );
+  }
+}
+
+@connect(null, { selectRejectedExperiment })
+@autobind
+export class ExperimentRejectButton extends React.PureComponent {
+  static propTypes = {
+    experiment: pt.object.isRequired,
+    selectRejectedExperiment: pt.func.isRequired,
+  };
+
+  handleClick() {
+    this.props.selectRejectedExperiment(this.props.experiment);
+  }
+
+  render() {
+    return (
+      <Button type="danger" onClick={this.handleClick}>Reject</Button>
+    );
+  }
+}
+
+
+export class ExperimentButtons extends React.PureComponent {
+  static propTypes = {
+    experiment: pt.object.isRequired,
+  };
+
+  render() {
+    return (
+      <div>
+        <ExperimentAcceptButton experiment={this.props.experiment} />
+        <ExperimentRejectButton experiment={this.props.experiment} />
+      </div>
+    );
+  }
+}
+
+
+export class ExperimentTitle extends React.PureComponent {
+  static propTypes = {
+    experiment: pt.object.isRequired,
+  };
+
+  render() {
+    const { experiment } = this.props;
+
+    return (
+      <div>
+        <span className="project-name">{experiment.project_name}:</span>
+        <span className="experiment-name">{experiment.name}</span>
+      </div>
+    );
+  }
+}
+
+
+export class Experiment extends React.PureComponent {
+  static propTypes = {
+    experiment: pt.object.isRequired,
+  };
+
+  render() {
+    const experiment = this.props.experiment;
+    const control = experiment.control;
+    const variant = experiment.variant;
+
+    const experimentButtons = <ExperimentButtons experiment={experiment} />;
+    const experimentTitle = <ExperimentTitle experiment={experiment} />;
+
+    return (
+      <div>
+        <Card
+          title={experimentTitle}
+          extra={experimentButtons}
+        >
+          <div className="experiment-section">
+            <div className="experiment-population">
+              <span className="experiment-population-size">
+                {parseFloat(experiment.population_percent)}%
```

Comment:
It's because the Experimenter API stores and serializes percents as fixed precision Decimals so it's transmitted as a string so you don't wind up getting something like 90.50000001 of Release as a configuration when you meant exactly 90.5%, but it looks ugly to show 90.5000% when you only need to say 90.5%.